assets/js/vendor/GoogleChart.js

```diff
@@ -179,7 +179,8 @@
     GoogleChart.prototype.bind = function(event, callback) {
       var _base;
 
-      (_base = this._listers)[event] || (_base[event] = this._addGoogleListener(event));
+      _base = this._listers;
+      _base[event] = this._addGoogleListener(event);
```

Comment:
I'm skeptical of this - I think the left side of the `||` is to only have one googleListener per event, whereas this will add a new listener every time. I'm not sure the ramifications of that.